src/index.ts

```diff
@@ -48,13 +52,42 @@ export async function initialise(options: InitOptions): Promise<State> {
     const validUcans = ucan.validatePermissions(permissions, authedUsername)
 
     if (validSecrets && validUcans) {
-      return scenarioContinuation(permissions, authedUsername, await maybeLoadFs(authedUsername))
+      return scenarioContinuation(
+        permissions,
+        authedUsername,
+        await maybeLoadFs(authedUsername)
+      )
     } else {
       return scenarioNotAuthorised(permissions)
     }
 
+  } else if (authedUsername && useWnfs) {
+    if (options.loadFileSystem === false) throw new Error("Must load filesystem when using the useWnfs option.")
```

Comment:
Using WNFS only makes sense when loading the filesystem.

Reply:
I'm not entirely sure about having this check here. I would remove it for flexibility. If I wanted to load my filesystem in a web worker instead, I'd use the `loadFileSystem` option, I imagine we want to still do that with `useWnfs`?

Reply:
Ah right, great point! Forgot about that use case. And indeed, it is more flexible to be able to load the filesystem after initialization. 👌